openAIKey.py

```diff
@@ -0,0 +1,9 @@
+import os
```

Comment:
Again, we should not need this. I would suggest removing and finding a local solution to point the interpreter to the right env.

Reply:
Will reply to this in the following mail

Reply:
@ytehran please address this (remove) in order to be able to merge into `main` :)